tests/dummy/app/templates/application.hbs

```diff
@@ -1,4 +1,4 @@
-{{osf-navbar loginAction=(action 'login')}}
+{{new-osf-navbar loginAction=(action 'login') currentService='PREPRINTS'}}
```

Comment:
Replaced old navbar with new navbar. I wasn't sure if we wanted to keep it, and it was a good reference while I was building the new one. 

Second, I haven't come up with a good idea to identify the consuming service, but don't know if it's needed right now.  We'll just be using this in two places:
{{osf-navbar loginAction=(action 'login') currentService='PREPRINTS'}}
{{osf-navbar loginAction=(action 'login') currentService='REGISTRIES'}}.

I've heard that we might be moving preprints and registries into one repo which would probably change how the service needs to be detected as well.